clang/test/dpct/helper_files_ref/include/blas_utils.hpp

```diff
@@ -385,6 +385,56 @@ inline void rk_impl(cl::sycl::queue &q, oneapi::mkl::uplo uplo,
   }
 }
 
+template <class Ta, class Tb, class Ts>
+inline void
+trsm_batch_impl(cl::sycl::queue &q, oneapi::mkl::side left_right,
+                oneapi::mkl::uplo upper_lower, oneapi::mkl::transpose trans,
+                oneapi::mkl::diag unit_diag, int m, int n, const void *alpha,
+                const void **a, int lda, void **b, int ldb, int batch_size) {
+  struct mat_info_t {
```

Comment:
maybe: math_info_t  ? Not sure, what "mat" means.

Reply:
"mat" means "matrix"

Reply:
If it is not common abbreviation/short form, then may be use the full form, like matrix_info_t

Reply:
OK